### DISCUSSION TITLE
Welcome to diabetes_prediction-AIF-project- Discussions!

### DISCUSSION THREAD

**Discussion by shivamoop** [shivamoop/diabetes_prediction-AIF-project-#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HussainM19** [comment]
What would be the most important factor while developing a model for diabetes prediction?
